Server/Server.cpp

```diff
@@ -1,44 +1,152 @@
-#include <iostream>
 #include "Server.hpp"
 
-Server::Server() {}
+Server::Server(const char *port) : m_servsock(-1) {
```

Comment:
dont't initialize server in constructer create **init()** function and initialize there